src/MSLGenerator.h

```diff
@@ -88,10 +88,6 @@ class MSLGenerator
     Target          m_target;
     
     bool            m_error;
-    
-    // TODO: Temporary hack to access uniforms struct members
-    HLSLBuffer* m_per_pass_buffer;
```

Comment:
Splitting uniform parameters in two buffers based on the frequency of change is critical for performance. I would imagine not every game cares about this, or would want to do it the same way, but I would certainly like to keep this feature, at least as an optional code path.